test/functorch/test_control_flow.py

```diff
@@ -1174,6 +1204,175 @@ def fwbw(map_op, f, x, y):
         fake_outs = fwbw(_fake_map, f, x, y)
         self.assertEqual(true_outs, fake_outs)
 
+    @unittest.skipIf(not SM70OrLater, "triton")
+    @unittest.skipIf(not torch.cuda.is_available(), "Test requires CUDA.")
+    @parametrize("device", [torch.device("cuda")])
+    def test_pointwise_associative_scan_vmap(self, device):
```

Comment:
@bohnstingl Could you also add a test that:
1) adds associative_scan to the hop_db (https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/hop_db.py). This involves adding some "sample inputs" to associative scan
2) calls opinfo_vmap_test? (https://github.com/pytorch/pytorch/blob/fb26b843906bbad5e28d1edccf298c74b8e00492/test/functorch/test_vmap.py#L4272C14-L4272C30)

That would be the ultimate vmap test